pkg/controller/pxcbackup/deadline.go

```diff
@@ -60,6 +65,44 @@ func checkStartingDeadline(ctx context.Context, cluster *api.PerconaXtraDBCluste
 	return errStartingDeadlineExceeded
 }
 
+func (r *ReconcilePerconaXtraDBClusterBackup) checkRunningDeadline(ctx context.Context, cluster *api.PerconaXtraDBCluster, cr *api.PerconaXtraDBClusterBackup) error {
+	log := logf.FromContext(ctx)
+
+	// check only if the current state is 'Starting'
+	if cr.Status.State != api.BackupStarting {
+		return nil
+	}
+
+	var deadlineSeconds *int32
+	if cr.Spec.RunningDeadlineSeconds != nil {
+		deadlineSeconds = cr.Spec.RunningDeadlineSeconds
+	} else if cluster.Spec.Backup.RunningDeadlineSeconds != nil {
+		deadlineSeconds = cluster.Spec.Backup.RunningDeadlineSeconds
+	}
+
+	if deadlineSeconds == nil {
+		return nil
+	}
+
+	job, err := r.getBackupJob(ctx, cluster, cr)
+	if err != nil {
+		return err
```

Comment:
let's add some context when returning this error given that `getBackupJob` does not add something and it is called from multiple different flows.

Reply:
done